.github/renovate.json

```diff
@@ -0,0 +1,9 @@
+{
+  "automerge": true,
+  "enabledManagers": ["github-actions", "npm"],
+  "ignoreDeps": ["cypress"],
```

Comment:
## Dependency Dashboard Approval

I recommend you enable the Dependency Dashboard issue on this repository, and then set Cypress to "Dependency Dashboard Approval"-workflow. This way you only get Cypress updates if you click a checkbox in the issue.

I don't recommend that you ignore dependencies forever. 😉

Benefits:

- Get a nice Dependency Dashboard to get an overview of pending updates
- Pull in updates ahead of schedule if wanted via the dashboard
- Only get Cypress updates when you click the checkbox in the Dependency Dashboard

## Proposed configuration:

1. Drop `ignoreDeps` line
2. Add new `packageRules` array:
    ```json
    {
      "packageRules": [
        {
          "matchPackageNames": ["cypress"],
          "dependencyDashboardApproval": true
        }
      ]
    }
    ```
3. Note that adding `dependencyDashboardApproval` also enables the Dependency Dashboard. So we don't have to set `dependencyDashboard=true` in the config.

## Links to documentation

- [General docs about "Dependency Dashboard Approval"](https://docs.renovatebot.com/getting-started/use-cases/#dependency-dashboard-approval)
- [Docs for `dependencyDashboardApproval` config option](https://docs.renovatebot.com/configuration-options/#dependencydashboardapproval)
- [Docs for `matchPackageNames` config option](https://docs.renovatebot.com/configuration-options/#matchpackagenames)

Reply:
@JoshuaKGoldberg I have applied the `dependecyDashboardApproval` settings. Will remove if you think you won't need it.